cloudify_agent/tests/shell/commands/test_install_local.py

```diff
@@ -0,0 +1,130 @@
+#########
+# Copyright (c) 2015 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  * See the License for the specific language governing permissions and
+#  * limitations under the License.
+
+import getpass
+import json
+import os
+import tempfile
+import uuid
+import shutil
+
+from celery import Celery
+from mock import patch
+
+from cloudify.utils import LocalCommandRunner
+from cloudify_agent.tests import BaseTest, utils
+from cloudify_agent.tests.api.pm import only_ci
+from cloudify_agent.installer.config import configuration
+
+from cloudify_agent.tests.installer.config import mock_context
+
+
+class TestInstaller(BaseTest):
+
+    @classmethod
+    def setUpClass(cls):
+        cls._resources_dir = tempfile.mkdtemp(
```

Comment:
BaseTest already creates a top level temp work dir (and takes care of cleaning it) so perhaps it would be better to simply create the resources dir under it?